lib/metasploit/framework/require.rb

```diff
@@ -49,7 +49,7 @@ def self.optionally(name, without_warning)
       #
       # @return [void]
       def self.optionally_active_record_railtie
-        if ::File.exist?(Rails.application.config.paths['config/database'].first)
```

Comment:
Because `Rails::Paths::Path` doesn't handle nil correctly. Delete your `database.yml` file and you should see the original error.